__tests__/app.test.js

```diff
@@ -232,3 +232,60 @@ describe("/api/articles/:article_id/comments", () => {
     });
   });
 });
+
+describe("/api/articles/:article_id/comments", () => {
+  describe("POST", () => {
+    test("201: Should take a username and body property and respond with the posted comment", () => {
+      return request(app)
+        .post("/api/articles/2/comments")
+        .send({ username: "butter_bridge", body: "coolio" })
+        .expect(201)
+        .then(({ body }) => {
+          expect(body.comment).toEqual({
+            body: "coolio",
+            votes: 0,
+            author: "butter_bridge",
+            article_id: 2,
+            comment_id: expect.any(Number),
+            created_at: expect.any(String),
+          });
+        });
+    });
```

Comment:
so *technically* you can't be sure that the comment has correctly been saved in the db with this (maybe the model does something weird, but still returns a 201).

we can test for this fully by doing a db.query in another .then block in the test which looks for this new made comment